resources/python/install_source_only.sh

```diff
@@ -59,6 +60,14 @@ rm -Rf $source_dir/incremental/tests
 # into other processes, which we don't require. That module is compiled, so we'll delete it.
 rm -Rf $source_dir/twisted/python/_sendmsg.so
 
+# Compress all files
+cd $source_dir
+zip -q -r pkgs.zip ./*
+cd $build_dir
+
+# Remove files
+find $source_dir/* ! -name $package_filename ! -name $requirements_filename -maxdepth 1 -exec rm -rf {} +
```

Comment:
I wonder if that would do the job:

```suggestion
find $source_dir/* ! -name $package_filename ! -name $requirements_filename -delete
```

Reply:
that leaves empty directories all around. Not useful.